bin/9l

```diff
@@ -40,7 +40,7 @@ case "$tag" in
 	userpath=true
 	extralibs="$extralibs -lutil -lresolv"
 	case "${SYSVERSION:-`uname -r`}" in
-	2.6.* | 3.* | 4.*)
+	2.6.* | 3.* | 4.* | 5.*)
```

Comment:
Might as well be ready for future releases:
```
	2.6.* | [3-9].* | [1-9][0-9].*)
```

Reply:
Thanks, done.